app/javascript/components/CoverPage.js

```diff
@@ -21,20 +21,48 @@ class CoverPage extends React.Component {
 
     let page_title;
     let cover_subtitle;
-    let cover_description_wrapper;
     let cover_description;
     let atlas_button;
+    let atlas_flex_div;
+    let logo_3_img_div;
+    let hr;
+    let cover_title_para;
+    let cover_description_para;
+    let cover_page_wrapper; 
+    let cover_page_wrapper_envelope ;
+
+
+
+
 
     if (siteTemplateName != 'INDIA') {
+
       page_title = <h2 className="cover-title"> {page.content_type == 3 ? current.name : page.name} </h2>;
       cover_subtitle = page.content_type !== 3 && <h2 className="cover-subtitle"> {current.name} </h2>;
+    
     } else {
       if (page.content_type == 3) {
 
-        page_title = <h2 className="cover-title-left">{current.name} </h2>;
         cover_description = <div className="cover-desc-wrapper" > <h2 className="cover-description"> {page.description} </h2> </div>;
+
         atlas_button = <div className="button_wrapper"> <a href="/atlas" className="home_page_atlas_button"> Launch Atlas </a> </div>;
-        cover_description_wrapper = <div className="cover-elem-wrapper"> {cover_description} {atlas_button} </div>;
+
+        atlas_flex_div = <div className="atlas_flex_div"> {atlas_button} </div>;
+
+        logo_3_img_div = <div className="restoration_image_div"> <img className="restoration_image" alt="Logo" src="http://wri-sites.s3.amazonaws.com/ifmt/NationalAtlasPartnerLogo/Atlas/RO_Atlas_white_png.png" /> </div>;
+
+        hr = <hr className="hr-homepage" />;
+
+        cover_title_para = <p className="cover-title-para"> {current.name} </p>;
+
+        cover_description_para = <p className="cover-description-para"> {page.description} </p>;
+
+        cover_page_wrapper = <div className="cover-page-wrapper"> {logo_3_img_div} {hr} {cover_title_para} {cover_description_para} </div>;
+
+        cover_page_wrapper_envelope = <div className="cover_page_wrapper_envelope"> {cover_page_wrapper} </div>
```

Comment:
Instead of putting each element in its own variable, in JSX youre allowed to do this @speppy 

```jsx
cover_page_wrapper_envelope = (
  <div className="cover_page_wrapper_envelope"> 
    <div className="cover-page-wrapper">
      <div className="restoration_image_div">
        <img className="restoration_image" alt="Logo" src="http://wri-sites.s3.amazonaws.com/ifmt/NationalAtlasPartnerLogo/Atlas/RO_Atlas_white_png.png" />
      </div>
      
      <hr className="hr-homepage" />

      <p className="cover-title-para"> {current.name} </p>
      <p className="cover-description-para"> {page.description} </p>

      <div className="atlas_flex_div">
      <div className="button_wrapper"> <a href="/atlas" className="home_page_atlas_button"> Launch Atlas </a> </div>
      </div>


    </div>
  </div>
)
```

This will make this a lot easier to read and also allow bugs in the feautre. when you change, double check that i put the items in the correct order.